deploy/src/main/deploy/resource-descriptors/hono-service-device-registry/hono-service-device-registry-deployment.yaml

```diff
@@ -73,8 +73,8 @@ spec:
       - command:
         - sh
         - -c
-        - cp -u /tmp/hono/example-credentials.json /var/lib/hono/device-registry/credentials.json;
-          cp -u /tmp/hono/example-tenants.json /var/lib/hono/device-registry/tenants.json
+        - cp -n /tmp/hono/example-credentials.json /var/lib/hono/device-registry/credentials.json;
```

Comment:
this will not work because the `cp` command in the busybox image does not support the `-n` switch:
```
/tmp # cp -n test test2
cp: invalid option -- n
BusyBox v1.30.0 (2018-12-31 18:16:17 UTC) multi-call binary.

Usage: cp [OPTIONS] SOURCE... DEST

Copy SOURCE(s) to DEST

	-a	Same as -dpR
	-R,-r	Recurse
	-d,-P	Preserve symlinks (default if -R)
	-L	Follow all symlinks
	-H	Follow symlinks on command line
	-p	Preserve file attributes if possible
	-f	Overwrite
	-i	Prompt before overwrite
	-l,-s	Create (sym)links
	-T	Treat DEST as a normal file
	-u	Copy only newer files
```